themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -186,4 +186,6 @@
   <link href="https://addtocalendar.com/atc/1.5/atc-style-blue.css" rel="stylesheet" type="text/css">
   {{- end -}}
 
+  <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
```

Comment:
I am pretty sure you do not need this on every page. You have it already included in `themes/digital.gov/static/workflow/index.html` a that should suffice ❤️

Reply:
Hey @jeremyzilar 👋  - no you don't need it on every page, but typically you would need it on the homepage since the widget redirects to the root url when it's getting the auth token. Luckily for us it looks like Federalist provides a custom integration that takes care of the redirect instead, so we only need it on the CMS page 👍 .